### PR TITLE
sni: new hint 'http2_initial_window_size_in'

### PR DESCRIPTION
**Pull request by randall**
Allow setting the H2 initial window size per SNI

### CODE REVIEW

**Comment by randall** [comment]
I'll rebase this soon.

**Comment by randall** [comment]
[approve ci autest]

**Comment by randall** [comment]
@bryancall Can I get a review here?

File: doc/admin-guide/files/sni.yaml.en.rst [hunk 1]
```diff
@@ -77,181 +77,187 @@ inbound_port_range        Inbound   The port range for the inbound connection in
...
+
+verify_client_ca_certs       Both      Specifies an alternate set of certificate authority certs to use to verify the
+                                       client cert.  The value must be either a file path, or a nested set of key /
+                                       value pairs.  If the value is a file path, it must specify a file containing the
+                                       CA certs.  Otherwise, there should be up to two nested pairs.  The possible keys
+                                       are ``file`` and ``dir``.  The value for ``file`` must be a file path for a file
+                                       containing CA certs.  The value for ``dir`` must be a file path for an OpenSSL
+                                       X509 hashed directory containing CA certs.  If a given file path does not begin
+                                       with ``/`` , it must be relative to the |TS| configuration directory.
+                                       ``verify_client_ca_certs`` can only be used with capabilities provided by
```
Comment (bneradt) [inline doc/admin-guide/files/sni.yaml.en.rst:130]: Thanks for fixing the misspellings.

**Review by bneradt — APPROVED** [review]
Sorry for the delay in review. Looks great to me. Thank you for adding this. I could see why this would be helpful.

**Pull request merged**